syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -0,0 +1,73 @@
+import torch
+import syft
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+
+from typing import Union
+
+
+class FalconHelper:
+    XOR_EXPECTED_RING_SIZE = 2
+
+    @classmethod
+    def unfold(cls, image, kernel_size, padding):
+        return cls.__switch_public_private(
+            image, cls.__public_unfold, cls.__private_unfold, kernel_size, padding
+        )
+
+    @staticmethod
+    def __public_unfold(image, kernel_size, padding):
+        image = image.double()
+        image = torch.nn.functional.unfold(image, kernel_size=kernel_size, padding=padding)
+        image = image.long()
+        return image
+
+    @staticmethod
+    def __private_unfold(image, kernel_size, padding):
+        return image.unfold(kernel_size, padding)
+
+    @staticmethod
+    def xor(
+        value: Union[int, ReplicatedSharingTensor], other: Union[int, ReplicatedSharingTensor]
+    ) -> ReplicatedSharingTensor:
+        """
+        Compute the XOR value between value and other
```

Comment:
please remove the docstring, xor is self-documented it doesn't need comments, "you should see every comment as a failure to express your intuition with code"

Reply:
We need to do docstring because if we move to ```syft_0.3.0``` that will fail if we do not have every function documented.